networks/mainnet/gentx/gentx-smartstake.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Smart Stake","identity":"DD06F013A474ACA3","website":"smartstake.io","security_contact":"info@smartstake.io","details":"Your hub for validator, network, and decentralization analytics. Support @ t.me/SmartStake. Like our services? Delegate on any of the networks listed at smartstake.io"},"commission":{"rate":"0.050000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"migaloo17jxt5pmmfaucelvh638c94zw0jd83j5m3477es","validator_address":"migaloovaloper17jxt5pmmfaucelvh638c94zw0jd83j5mry6jmm","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"WDPhhqeYu93/1Ay9sRQlaCPICWZByW8GVLyWON3h+iA="},"value":{"denom":"uwhale","amount":"10000000"}}],"memo":"1cd714435bf9fdfcaa5f4461896b28b6a97ad5a7@88.208.241.28:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A2fu8HlDqmTw2q1N3w9/tREqGVMjQ0jMJT5NPvuz140x"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["TXmZm5X19J5FGDTwe0RiL1TcuDGJIsO3aVxscSj5Xmx8n+CFw78dbeoYqYpjEhwJiB6NOje67HYiBiTAF700lw=="]}
```

Comment:
Your validator, and your rules of course, but I'd set a higher max rate (fwiw)

Reply:
heh you have many validators.  Will resolve and approve.

Reply:
> Your validator, and your rules of course, but I'd set a higher max rate (fwiw)

it's smart stake's standard max rate

Reply:
> heh you have many validators. Will resolve and approve.

i believe the previous PR had testnet gentx. the new file today is the only one for mainnet